### PR TITLE
carusel

### PR DESCRIPTION
**Pull request by 006080**
I put SharePopup into the right folder and added a carousel of images.

### CODE REVIEW

**Comment by 006080** [comment]
Conflict solved

**Comment by Konzisam** [comment]
there seems to be conflicts

File: src/components/ProductCard/ProductCard.css
Comment (Konzisam) [inline src/components/ProductCard/ProductCard.css:13]: We dont need this unless as it is not in the airbnb,unless  you just like the effect and oyu want to keep it:)

**Review by Konzisam — CHANGES_REQUESTED** [review]
1. you seemd to have chenged the css file form module.css to .css, do oyu have  reason for that.... because we had agreed to use modular css. 
2. also you remove the router link in the product card unnecesarilly
3. in the products card, the arrow right and left work, but they dont really look like the ones in th airbnb ...(**low priority: I would approve if the above are fixed**)

**Comment by miklesch** [comment]
Hi, 
beside the last comment of Konzisam I  think it is essential to merge main into your branch as there have been so many changes...

**Comment by 006080** [comment]
Hi. team. I am a bit in doubt... I was resolving the conflicts, but after reviewing the project, I noticed that we already have a server-side setup in place. Will be better if we integrate imediately the new destinations JSON file into the server part? 
Or I will first solve a conflict in the current branch?
Let me know if you have any concerns or suggestions! Thanks

**Comment by Konzisam** [comment]
They are two branches so you need to deal with them independently. 
Conflicts only involve your working branch and main branch and  arise only when something in main is on the same 'line' as something in your branch, git gets confused which one to choose. 
Since the branch with json file isn't merged to main it wont affect anything, it actually won't be a conflict in the first place (even if it was merged) because there is no any other destination json file in main. 
Would suggest just fixing the issues in this specific branch. 
Hope this kind of clears the doubts.

**Comment by 006080** [comment]
Thank you for the clarification, but I believe that's quite obvious to
anyone. My question was whether it might be better to wait for the JSON
file branch to be merged. I honestly don’t understand what the issue is
with adding this PR. There’s only one file, with no conflicts. Is it really
that difficult to approve it?

**Comment by 006080** [comment]
As I have to move ahead with creating a grid using a Product.js Component I solved conflicts in a current file. I'll wait for approval if it looks good.
Then if it works I will change CSS to modular CSS.

File: src/App.jsx
Comment (miklesch) [inline src/App.jsx:90]: The ProductCard component is used twice (also on line 56). Is this on purpose?
The Link here is the old version. it´s currently different on main.

File: src/components/ProductCard/ProductCard.jsx
Comment (miklesch) [inline src/components/ProductCard/ProductCard.jsx:38]: is used twice, also at line 6

**Review by miklesch — COMMENTED** [review]
when i start this branch on my machine, i am getting following error: 
[cause]: Error: Cannot find module '@rollup/rollup-win32-x64-msvc'
  Require stack:
  - F:\Coding\Web_Circle\F24-Berlin-Web-Circle\node_modules\rollup\dist\native.js
  
when i update npm, there are still a lot of problems at the productCard component.
please check or leave me a message, if I miss something.

**Comment by 006080** [comment]
Currently I am working on this branch. Will push the changes very soon.

**Comment by 006080** [comment]
Screenshot from 2024-10-16 17-35-20

Here is how a Modal looks now.

**Comment by 006080** [comment]
Screenshot from 2024-10-16 18-08-29

Here is how a ProductCard looks now :sunglasses:

**Comment by 006080** [comment]
Screenshot from 2024-10-16 18-53-09
SharePopup Modal with overlay.

**Comment by 006080** [comment]
I also added dots to the carousel and overlay

**Review by Konzisam — APPROVED** [review]

**Comment by Konzisam** [comment]
looks good! Kudos Nataliya

**Pull request merged**
